bentoml/_internal/frameworks/keras.py

```diff
@@ -154,7 +154,10 @@ def save(
     Examples::
     """  # noqa
     tf.compat.v1.keras.backend.get_session()
-    context: t.Dict[str, t.Any] = {"tensorflow": _tf_version}
+    context: t.Dict[str, t.Any] = {
+        "framework_name": "tensorflow",
+        "pip_dependencies": [f"tensorflow=={_tf_version}"],
+    }
```

Comment:
should `framework_name` be `"keras"` or `"tensorflow"`. I prefer keras

Reply:
yes, Keras makes more sense. Also Keras is moving its development back to its own repo